tests/nlu/classifiers/test_diet_classifier.py

```diff
@@ -49,6 +52,120 @@
 from rasa.utils.tensorflow.model_data_utils import FeatureArray
 
 
+@pytest.fixture()
+def default_diet_resource() -> Resource:
+    return Resource("DIET")
+
+
+@pytest.fixture()
+def create_diet(
```

Comment:
I like the fixture+inner constructs :) Except this one, that looks like a a complicated way of having Resource("DIET") as the default 😅

Reply:
Without it we would also have to pass in `default_model_storage` and `default_execution_context` everytime.

Reply:
Hm, `model_storage` and `default_execution_contexts` could also have defaults? ... But I see that one day we might want to test different storages/contexts, so that'll be handy then :)